### ISSUE TITLE
[ESP32S3] v4.4 deep_sleep sleeping more than should (IDFGH-10088)

### ISSUE THREAD

**Issue by KonssnoK** [espressif/esp-idf#11363]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### General issue report

issue is quite simple:
- we put 30 minutes sleep, device sleeps 40 minutes
- we put 1 hour sleep, device sleeps 1.5hours
- we put 1.5 hours sleep, device sleeps 3hours

Any idea on how to improve accuracy?

In the settings
```
- 150000 Hz if internal RC oscillator is used as clock source. For this use value 1024. 
- 32768 Hz if the 32k crystal oscillator is used. For this use value 3000 or more. 
- In case more value will help improve the definition of the launch of the crystal. 
- If the crystal could not start, it will be switched to internal RC.
```
should we consider the 8MHz as a 32kHz?

our RTC settings:
![image](https://user-images.githubusercontent.com/10288926/237028008-11f3e4b1-55b7-4801-a298-2da7af485f70.png)

**Comment by 10086loutianhao** [comment]
I will take a look at this issue

**Comment by 10086loutianhao** [comment]
I see that the latest master branch has resolved this issue. Indeed, when using 8Md256, the improper PMU parameters caused abnormal counting in the lp_timer. The more appropriate parameters have been implemented on the latest master branch. (The cause of the malfunction is the same as in GH#11362)

**Comment by 10086loutianhao** [comment]
> @10086loutianhao was this merged in v4.4?
Yes, the solution here should be the same as the one in https://github.com/espressif/esp-idf/issues/11362. Please check it.
